Formula/idris2.rb

```diff
@@ -15,6 +15,7 @@ class Idris2 < Formula
 
   depends_on "chezscheme"
   depends_on "coreutils"
+  depends_on "gmp"
```

Comment:
Do we know if this is needed at runtime?

Reply:
I'm not sure. I assume the difference is whether it's statically vs dynamically linked, right? I will ask the devs.

Reply:
I had a look; it doesn't appear to be dynamically linked.